ngbootbox/ngbootbox.d.ts

```diff
@@ -0,0 +1,38 @@
+// Type definitions for ngbootbox
+// Project: https://github.com/eriktufvesson/ngBootbox
+// Definitions by: Sam Saint-Pettersen <https://github.com/stpettersens>
+// Definitions: https://github.com/borisyankov/DefinitelyTyped
+
+/// <reference path="../es6-promise/es6-promise.d.ts" />
+/// <reference path="../bootbox/bootbox.d.ts" />
+
+interface IBootboxDialog {
```

Comment:
http://www.typescriptlang.org/Handbook#writing-dts-files

> Naming Conventions
In general, do not prefix interfaces with I (e.g. IColor). Because the concept of an interface in TypeScript is much more broad than in C# or Java, the IFoo naming convention is not broadly useful.